### PR TITLE
fixes all known autoseed issues

### PR DESCRIPTION
**Pull request by ProofOfKeags**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-  # Cleanup
-  && apt-get autoremove -y \
-  && apt-get clean \
-  && rm -rf /var/lib/apt/lists/*
+RUN apt-get update
+RUN apt-get install -y mariadb-server wget sqlite3 apache2-utils curl
+# Cleanup
+RUN apt-get autoremove -y
+RUN apt-get clean
+RUN rm -rf /var/lib/apt/lists/*
```
Comment (elvece) [inline Dockerfile:9]: why separate `RUN` commands?

**Review by chrisguida — APPROVED** [review]
utACK

File: manifest.yaml [hunk 2]
```diff
@@ -28,6 +28,16 @@ main:
     main: /media
     filebrowser: /mnt/filebrowser
   io-format: yaml
+migrations:
+  from:
+    "<2.3.9":
+      type: docker
+      image: main
+      entrypoint: sh
+      args: ["/usr/local/bin/migration-from-lt-2-3-9.sh"]
```
Comment (elvece) [inline manifest.yaml:37]: you do not need to prefix with `/usr/local/bin`

Comment (elvece) [inline manifest.yaml:37]:
```suggestion
      entrypoint: migration-from-lt-2-3-9.sh
      args: []
```

File: Dockerfile (hunk 1 above)
Comment (ProofOfKeags) [reply Dockerfile:9]: This was helping me debug the segfaulting issues. It's not necessary to leave in there as is, I can remove it if you want.

Comment (elvece) [reply Dockerfile]: Gotcha, I think its more efficient to include them as a single line so as to not create a separate layer.

**Review by elvece — APPROVED** [review]

**Pull request merged**